spec/features/openid_connect/redirect_uri_validation_spec.rb

```diff
@@ -191,8 +205,8 @@ def visit_idp_from_sp_with_ial1_with_invalid_redirect_uri(state: SecureRandom.he
     )
   end
 
-  def visit_idp_from_inactive_sp(state: SecureRandom.hex)
-    client_id = 'inactive'
+  def visit_idp_from_nonexistant_sp(state: SecureRandom.hex)
+    client_id = 'nonexistant:sp'
```

Comment:
Spelling nit:
```suggestion
  def visit_idp_from_nonexistent_sp(state: SecureRandom.hex)
    client_id = 'nonexistent:sp'
```

Reply:
Good catch, I'm not sure why I didn't see/IDE didn't bug me about that.